tests/pfcwd/test_pfcwd_all_port_storm.py

```diff
@@ -234,8 +251,9 @@ def test_all_port_storm_restore(self, duthosts, enum_rand_one_per_hwsku_frontend
                 test_port = device_conn[intf]['peerport']
                 if test_port in setup_pfc_test['test_ports']:
                     selected_test_ports.append(test_port)
-
-        with send_background_traffic(duthost, ptfhost, queues, selected_test_ports, setup_pfc_test['test_ports']):
+        resolve_arp(duthost, ptfhost, setup_pfc_test['test_ports'])
+        with send_background_traffic(duthost, ptfhost, queues, selected_test_ports, setup_pfc_test['test_ports'],
+                                     pkt_count=500):
```

Comment:
May I know why the packet count is changed to 500? It can introduce some issue since the packet count was 100000 before this change

Reply:
The background traffic is sending per port one by one, if send too much packet per port, it may increase the packet sending period, which may lead to the background traffic failed to be deteced at DUT in detect period.

Reply:
I still have concern on this change. Sending 500 background packets may not be enough.

Reply:
@echuawu Can you address the comment? I don't think sending 500 background packets is good enough. It's possible that the background traffic sending is done, while PFCWD is not triggered.